src/gval/statistics/continuous_statistics.py

```diff
@@ -59,7 +59,7 @@ def __init__(self):
                 "Union[xarray.core.dataarray.DataArray, xarray.core.dataset.Dataset]",
                 "Union[xarray.core.dataset.Dataset, xarray.core.dataarray.DataArray]",
             ],
-            "return_type": [Number],
+            "return_type": [float, Number],
```

Comment:
Is float not a Number?

Reply:
Yes it is, but this gives the user the freedom to specifically constrain the function to only return float if they want that option.  If they use Number any number type would suffice.  Checking for return type in signature validation is literal and shouldn't check for parent class or subclass.